src/js/ui/function.js

```diff
@@ -0,0 +1,92 @@
+// eslint-disable-next-line no-unused-vars
+import { VENDING_MACHINE_CONSTANT } from '../service/constant.js';
+import { ELEMENT, querySelector } from './element.js';
+
+const inputName = querySelector(ELEMENT.INPUT.PRODUCT_NAME);
+const inputPrice = querySelector(ELEMENT.INPUT.PRODUCT_PRICE);
+const inputAmount = querySelector(ELEMENT.INPUT.PRODUCT_AMOUNT);
+const inputChargeAmount = querySelector(ELEMENT.INPUT.CHARGE_AMOUNT);
+/**
+ *
+ * @param {import('../service/vendingmachine.js').VendingMachine} vendingMachine
+ */
+export const addProduct = (vendingMachine) => {
+  const [name, price, amount] = [inputName.value, Number(inputPrice.value), Number(inputAmount.value)];
+  vendingMachine.addItem({ name, price, amount });
+};
+
+/**
+ *
+ * @param {import('../service/vendingmachine.js').VendingMachine} vendingMachine
+ */
+export const insertCoins = (vendingMachine) => {
+  vendingMachine.insertCoins(Number(inputChargeAmount.value));
+};
+
+/**
+ *
+ * @param {HTMLElement} element
+ */
+const clearInput = (inputElement) => {
+  inputElement.value = '';
+};
+
+export const clearProductInputs = () => {
+  [inputName, inputPrice, inputAmount].forEach((input) => clearInput(input));
+};
+
+export const clearChargeAmountInput = () => {
+  clearInput(inputChargeAmount);
+};
+
+/**
+ *
+ * @param {import('../service/vendingmachine.js').VendingMachine} vendingMachine
+ */
+export const renderProduct = (vendingMachine) => {
+  const products = vendingMachine.getProducts();
+  const innerHTML = products.reduce((result, { name, price, amount }) => {
+    const row = `<tr>
+      ${[name, price, amount].reduce((result, prop) => result + `<td>${prop}</td>`, '')}
+    </tr>`;
+    return result + row;
+  }, '');
+  querySelector(ELEMENT.TABLE.VENDING_MACHINE_PRODUCT_TBODY).innerHTML = innerHTML;
```

Comment:
```suggestion
  querySelector(ELEMENT.TABLE.VENDING_MACHINE_PRODUCT_TBODY).innerHTML = products.map({ name, price, amount }) => `
    <tr>
      <td>${name}</td>
      <td>${price}</td>
      <td>${amount}</td>
    </tr>
  `).join('');
```
이럴 땐 map으로 표현하는게 더 적합한 것 같아요!